translations/en/LC_MESSAGES/messages.po

```diff
@@ -19,49 +19,49 @@ msgstr ""
 "Generated-By: Babel 2.14.0\n"
 
 msgid "Access Before Assign"
-msgstr "We detected that a variable `{name}` is being used on line {access_line_number} before being set. Can you set the variable before it is used?"
+msgstr "We detected that a variable `{name}` is being used on line {access_line_number} before being set. Can you set the variable first?"
 
 msgid "Cyclic Var Definition"
-msgstr "We detected that a variable `{variable}` is being used on the right-hand side of the `{is}` command before being set. Can you set the variable before it is used?"
+msgstr "We detected that a variable `{variable}` is being used on the right-hand side of the `{is}` before being set. Can you set the variable first?"
 
 msgid "Else Without If Error"
 msgstr "We detected that there is an `{else}` being used before an `{if}` on line {line_number}. Can you try writing an `{if}` before the `{else}`?"
 
 msgid "Function Undefined"
-msgstr "We detected that a function {name} is being used without being defined. Can you define the function before it is used?"
+msgstr "We detected that a function `{name}` is being used without being defined. Can you define the function first?"
 
 msgid "Has Blanks"
 msgstr "We detected that the code seems to be incomplete. Can you fill in the blanks with code?"
 
 msgid "Incomplete"
-msgstr "We detected that part of the code seems to be missing from the `{incomplete_command}` on line {line_number}. Can you try adding the what's missing?"
+msgstr "We detected that part of the code seems to be missing from the `{incomplete_command}`, on line {line_number}. Can you try adding what's missing?"
 
 msgid "Incomplete Repeat"
-msgstr "We detected that the `{repeat}` on line {line_number} is missing a `{command}` command. Can you try adding it?"
+msgstr "We detected that the `{repeat}` seems to be missing a `{command}` command, on line {line_number}. Can you try adding `{repeat}` with `{command}`?"
```

Comment:
The 'seems to be' part is a bit long and doen not add something perse. We could make it shorter without losing info?

Reply:
I have made a suggestion, also moving the line number into the sentence, which I find reads nicer? If you want to move it back, feel free to do so!